f/connectors/alerts/alerts_gcs.script.yaml

```diff
@@ -53,6 +55,21 @@ schema:
         will be retrieved. This ID is used to filter and process relevant
         alerts.
       default: null
+    territory_name:
```

Comment:
I'm not convinced on this variable name, as it is ambiguous. 

A community operator might misunderstand this to refer to the real name of their territory, not the GC subdomain slug e.g. `{slug}.guardianconnector.net`.

On the other hand, "slug" is not a commonly known term for non-technical users. 

Maybe "subdomain" is a better choice.